ariadne/src/Ariadne/Wallet/Knit.hs

```diff
@@ -278,3 +278,6 @@ mkPassPhrase = maybe emptyPassphrase (ByteArray.convert . hashRaw . encodeUtf8)
 
 getPassPhraseArg :: Elem components Core => ArgumentConsumer components PassPhrase
 getPassPhraseArg = mkPassPhrase <$> getArgOpt tyString "pass"
+
+getChainTypeArg :: Elem components Core => ArgumentConsumer components HdAddressChain
+getChainTypeArg = undefined
```

Comment:
It's questionable whether the wallet we are developing should be such advanced that generating an `internal` address manually is possible. Normally only `external` addresses can be generated manually and `internal` addresses are generated automatically.
I tend to think that even in advanced wallet generating `internal` addresses shouldn't be possible.

Reply:
And even if we want to permit generation of `internal` addresses, I'd make it a flag, so that `external` addresses are generated by default (because that's what normal people need) and `internal` needs to be specified explicitly.
`internal: true`, for example.